source/module_dati_fisici.f90

```diff
@@ -212,10 +212,10 @@ SUBROUTINE inizializza_dati_fisici(file_reticolo_opt)
 		ELSE IF ( crystal_cell=='hring' ) THEN
 
 			r_crystal(1:3,1)=(/ L_cov_bond		,	0.d0						,	0.d0/)
-			r_crystal(1:3,2)=(/ L_cov_bond*0.5d0,	sqrt(3.d0)*L_cov_bond*0.5d0	,	0.d0/)
-			r_crystal(1:3,3)=(/-L_cov_bond*0.5d0,	sqrt(3.d0)*L_cov_bond*0.5d0	,	0.d0/)
-			r_crystal(1:3,4)=(/-L_cov_bond		,	0.d0						,	0.d0/)
-			r_crystal(1:3,5)=(/-L_cov_bond*0.5d0,	-sqrt(3.d0)*L_cov_bond*0.5d0,	0.d0/)
+			r_crystal(1:3,4)=(/ L_cov_bond*0.5d0,	sqrt(3.d0)*L_cov_bond*0.5d0	,	0.d0/)
```

Comment:
Oh wow this change is 2 years old... I think I was thinking as a series of spin up/down pairs, 1+4, 2+5, 3+6. The way I built this the spin should be alternating around the H6 ring. But of course we can change the order of code lines according to 1,2,3,4,5,6.